README.md

```diff
@@ -118,6 +118,49 @@ Here is an example of the default file format that go-i18n supports:
 
 To use a different file format, write a parser for the format and add the parsed translations using [AddTranslation](https://godoc.org/github.com/nicksnyder/go-i18n/i18n#AddTranslation).
 
+Flat Format
```

Comment:
I think we need to update the `goi18n merge` command to have an option to output the flat format before we advertise this on the main readme. Up to you whether you want to just remove the README update from this PR or update the merge command.